detekt-core/src/test/kotlin/io/gitlab/arturbosch/detekt/core/CorrectableRulesFirstSpec.kt

```diff
@@ -19,14 +19,14 @@ class CorrectableRulesFirstSpec {
         var actualLastRuleId = ""
 
         class First(config: Config) : Rule(config) {
-            override val issue: Issue = justAnIssue.copy(id = "NonCorrectable")
+            override val issue: Issue = Issue("NonCorrectable", justAnIssue.description, justAnIssue.debt)
```

Comment:
```suggestion
            override val issue: Issue = Issue("NonCorrectable", "", Debt.FIVE_MINS)
```

and remove the variable `justAnIssue`